### ISSUE TITLE
style.setProperty doesn't seem to be working for custom element subclasses of CSSStyleDeclaration

### ISSUE THREAD

**Issue by pshaughn** [servo/servo#25013]
WPT custom-elements/reactions/CSSStyleDeclaration.html makes a custom element class based on "style" and tries to modify it in various ways to see what callbacks fire. Most of the tests fail because .setProperty() isn't working, not only in that callbacks aren't firing but in that the property isn't getting set. The way the tests are written, there may be other errors this one is masking.

**Comment by pshaughn** [comment]
I think #25130 changed this; most cases are passing, but style.webkitFilter and style['-webkit-filter' are still failing. Despite the browser-specific origins of such a case there is cross-browser support; https://wpt.fyi/results/custom-elements/reactions/CSSStyleDeclaration.html?label=experimental&label=master&aligned is solid green.

**Comment by jdm** [comment]
That looks like https://drafts.csswg.org/cssom/#ref-for-supported-css-property%E2%91%A2.

**Comment by pshaughn** [comment]
Yes, and just above there we see a similar case Servo does handle about camel-cased versions. components/style/properties/properties.mako.rs appears to be what's responsible for making these alternate versions of property names happen.
